src/me/mrCookieSlime/Slimefun/AncientAltar/RitualAnimation.java

```diff
@@ -82,7 +109,11 @@ private void checkPedestal(Block pedestal) {
 				e.printStackTrace();
 			}
 
+			
+			System.out.println("UnLocking item: " + item.getCustomName() + " / " + item.getItemStack().getType().name());
```

Comment:
Pretty sure that this debug message should be removed.

Reply:
Doh yeah, I’ll fix that